build-consumer.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
```

Comment:
Add a  `set -e` at the top of these scripts. That'll make the script fail and stop immediately instead of trying to keep running the commands if a prior one failed